PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,40 @@
 ## High-level description
 
+<!-- This template uses HTML comments for providing guidance. They are not visible in the PR output -->
+
+
 What features does this change enable? What bugs does this change fix?
 
 
-## Corresponding DC/OS tickets (obligatory)
+## Corresponding DC/OS tickets (required)
+
+<!--
+
+Please provide a list of JIRA tickets that this Pull Request acts upon. This section will be parsed, 
```

Comment:
Maybe this could be more succinct, possibly with a document for more information. I don't expect most readers of this will need to know the details of how fix version works.

Reply:
Let's keep the Fix Version information pointer. Fix Version update is relied upon, and brief mention or hyperlink could serve as reminder or side towards a safer approach than not mentioning it.